projects/cn0506_mii/zcu102/system_constr.xdc

```diff
@@ -0,0 +1,67 @@
+set_property -dict {PACKAGE_PIN AA11 IOSTANDARD LVCMOS18} [get_ports {mii_rxd_b[0]}]               ; ## H22 FMC_HPC1_LA19_P
+set_property -dict {PACKAGE_PIN AA10 IOSTANDARD LVCMOS18} [get_ports {mii_rxd_b[1]}]               ; ## H23 FMC_HPC1_LA19_N
+set_property -dict {PACKAGE_PIN AB11 IOSTANDARD LVCMOS18} [get_ports {mii_rxd_b[2]}]               ; ## G21 FMC_HPC1_LA20_P
+set_property -dict {PACKAGE_PIN AB10 IOSTANDARD LVCMOS18} [get_ports {mii_rxd_b[3]}]               ; ## G22 FMC_HPC1_LA20_N
+set_property -dict {PACKAGE_PIN AF10 IOSTANDARD LVCMOS18} [get_ports mii_tx_clk_b]                 ; ## G28 FMC_HPC1_LA25_N
+set_property -dict {PACKAGE_PIN AH12 IOSTANDARD LVCMOS18 SLEW FAST} [get_ports mii_tx_en_b]        ; ## H28 FMC_HPC1_LA24_P
+set_property -dict {PACKAGE_PIN AC12 IOSTANDARD LVCMOS18 SLEW FAST} [get_ports {mii_txd_b[0]}]     ; ## H25 FMC_HPC1_LA21_P
+set_property -dict {PACKAGE_PIN AC11 IOSTANDARD LVCMOS18 SLEW FAST} [get_ports {mii_txd_b[1]}]     ; ## H26 FMC_HPC1_LA21_N
+set_property -dict {PACKAGE_PIN AF11 IOSTANDARD LVCMOS18 SLEW FAST} [get_ports {mii_txd_b[2]}]     ; ## G24 FMC_HPC1_LA22_P
+set_property -dict {PACKAGE_PIN AG11 IOSTANDARD LVCMOS18 SLEW FAST} [get_ports {mii_txd_b[3]}]     ; ## G25 FMC_HPC1_LA22_N
+
+set_property -dict {PACKAGE_PIN T13  IOSTANDARD LVCMOS18 PULLUP true} [get_ports mdio_fmc_b]       ; ## H31 FMC_HPC1_LA28_P
+set_property -dict {PACKAGE_PIN R13  IOSTANDARD LVCMOS18} [get_ports mdc_fmc_b]                    ; ## H32 FMC_HPC1_LA28_N
+
+set_property -dict {PACKAGE_PIN AE9  IOSTANDARD LVCMOS18} [get_ports reset_b]                      ; ## H20 FMC_HPC1_LA15_N
+set_property -dict {PACKAGE_PIN AE10 IOSTANDARD LVCMOS18} [get_ports link_st_b]                    ; ## G27 FMC_HPC1_LA25_P
+set_property -dict {PACKAGE_PIN AF12 IOSTANDARD LVCMOS18} [get_ports mii_crs_b]                    ; ## D24 FMC_HPC1_LA23_N
+set_property -dict {PACKAGE_PIN AE12 IOSTANDARD LVCMOS18} [get_ports led_0_b]                      ; ## D23 FMC_HPC1_LA23_P
+set_property -dict {PACKAGE_PIN T12  IOSTANDARD LVCMOS18} [get_ports led_bl_c_c2m]                 ; ## D26 FMC_HPC1_LA26_P
+set_property -dict {PACKAGE_PIN R12  IOSTANDARD LVCMOS18} [get_ports led_bl_a_c2m]                 ; ## D27 FMC_HPC1_LA26_N
+set_property -dict {PACKAGE_PIN AG10 IOSTANDARD LVCMOS18} [get_ports led_br_c_c2m]                 ; ## G18 FMC_HPC1_LA16_P
+set_property -dict {PACKAGE_PIN AG9  IOSTANDARD LVCMOS18} [get_ports led_br_a_c2m]                 ; ## G19 FMC_HPC1_LA16_N
+
+create_clock -name rx_clk_1       -period   40.0 [get_ports mii_rx_clk_a]
+create_clock -name rx_clk_2       -period   40.0 [get_ports mii_rx_clk_b]
+create_clock -name tx_clk_1       -period   40.0 [get_ports mii_tx_clk_a]
+create_clock -name tx_clk_2       -period   40.0 [get_ports mii_tx_clk_b]
+create_clock -name ref_clk_125    -period    8.0 [get_ports ref_clk_125_p]
```

Comment:
This is not part of the design, should be removed.

Reply:
Done.